### ISSUE TITLE
针对聊天过程中，在知识库中未匹配到相应内容

### ISSUE THREAD

**Issue by Carlos-0924** [c121914yu/FastGPT#58]
是否能够增加自动记录板块，这边也便于了解知识库中有哪些问题未完善，能够快速完善知识库

**Comment by c121914yu** [comment]
这个逻辑不成立。如果没有匹配到，怎么能自动记录。只有匹配到，才能做操作。

**Comment by Carlos-0924** [comment]
未匹配到知识库有对应内容，记录用户的提问。然后跟根据记录的问题，补充完善知识库。无法这样操作吗？

**Comment by c121914yu** [comment]
首先，只能记录归属自己的聊天，如果是分享的聊天肯定不能记录。

第二，全自动记录可能会有很多测试数据被记录。

第三，很难判断是不是不存在这个知识点，还是模型没有理解。只有完全没有匹配的时候，才能判断。但是这个时候，你可以立马去看数据是否正确。

综合起来，这个需求更适合通过主动反馈的问题来解决。

**Comment by Carlos-0924** [comment]
是的。如果只是记录个人聊天内容，确实意义不大。除非是记录该应用或该知识库调用下所产生的所有聊天记录，然后对记录进行筛选。针对AI回答知识库无此内容的情况进行记录。
主动反馈也是一个不错的方法，对于分享链接出去给C端用户使用的情景下，在AI回答未匹配到相应内容的情况下，提供用户反馈功能。

**Comment by Carlos-0924** [comment]
是的。如果只是记录个人聊天内容，确实意义不大。除非是记录该应用或该知识库调用下所产生的所有聊天记录，然后对记录进行筛选。针对AI回答知识库无此内容的情况进行记录。
主动反馈也是一个不错的方法，对于分享链接出去给C端用户使用的情景下，在AI回答未匹配到相应内容的情况下，提供用户反馈功能。

**Comment by c121914yu** [comment]
日志，反馈以及标注功能已上线
